scripts/generator/graphql_generator/csharp/Cart.CheckoutAccessors.cs.erb

```diff
@@ -5,6 +5,19 @@ namespace <%= namespace %> {
     using <%= namespace %>.GraphQL;
 
     public partial class Cart {
+
+        /// <summary>
+        /// Retrieve the currency code for the current checkout.
+        /// </summary>
+        /// <exception cref="NoCheckoutException">Calling this accessor before a checkout was created for the Cart. See Cart.GetWebCheckoutLink</exception>
+        public CurrencyCode GetCurrencyCode() {
```

Comment:
I wonder if this should be:
```
public CurrencyCode? GetCurrencyCode()
```
Instead of thrown an exception we could return `null` if `CurrentCheckout == null`.

Another option would be to return `CurrencyCode.UNKNOWN` but I feel doing this would require a lot more domain knowledge than just simply returning `null`.